src/generic/gridctrl.cpp

```diff
@@ -630,7 +630,7 @@ void wxGridCellStringRenderer::Draw(wxGrid& grid,
             for (int i = col + cell_cols; i <= col_end; i++)
             {
                 clip.width = grid.GetColSize(i) - 1;
-                dc.DestroyClippingRegion();
+                wxDCClipper clipper(dc, clip);
                 dc.SetClippingRegion(clip);
```

Comment:
Sorry, this looks wrong as well, we shouldn't have to set it explicitly as `wxDCClipper` already just did it. No idea why was it preserved, the original patch did correctly remove it and so will I. Again, please let me know if I'm missing anything.